pkg/minishift/registration/utils.go

```diff
@@ -60,6 +66,10 @@ func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
 
 // Unregister host VM
 func UnregisterHostVM(host *host.Host, param *RegistrationParameters) error {
+	if param.SkipUnRegistration {
+		log.Debug("User choosed to skip un-registration")
```

Comment:
At this stage we should only log. 

@LalatenduMohanty If we want to better the user feedback, it should be in stop.go. The code would beed to check whether the VM is running and registered. If so and is stop is called with skip-unregistration, we can print a message. 

For my part I would be fine to merge, just with some updated log messages.